includes/migration/wc/class-fs-wc-migration.php

```diff
@@ -981,24 +765,24 @@ protected function get_purchase_vat_for_api() {
 		protected function get_purchase_for_api() {
 			$purchase                         = array();
 			$purchase['billing_cycle']        = 0;
-			$purchase['payment_method']       = $this->get_local_purchase_gateway();
-			$purchase['customer_external_id'] = 'edd_customer_' . $this->_edd_customer->id;
-			$purchase['license_key']          = self::$_edd_sl->get_license_key( $this->_edd_license->ID ); // Preserve the same keys.
+			$purchase['payment_method']       = $this->_wc_order->payment_method;
+			$purchase['customer_external_id'] = 'wc_customer_' . $this->ep->order->user_id;
+			$purchase['license_key']          = substr( $this->ep->order->api_key, 6 );
 			$purchase['license_quota']        = $this->get_license_quota(); // Preserve license activations limit.
-			$purchase['processed_at']         = $this->get_payment_process_date( $this->_edd_payment );
-			$purchase['payment_external_id']  = $this->get_payment_transaction_id( $this->_edd_payment );
+			$purchase['processed_at']         = $this->_wc_order->order_date;
+			$purchase['payment_external_id']  = $this->get_payment_transaction_id();
 
 			// Set license expiration if not a lifetime license via a purchase.
 			$license_expiration = $this->get_local_license_expiration();
 			if ( null !== $license_expiration ) {
 				$purchase['license_expires_at'] = $license_expiration;
 			}
 
-			if ( $this->local_is_sandbox_purchase() ) {
-				$purchase['is_sandbox'] = true;
+			if ( ! $purchase['payment_method'] ) {
```

Comment:
When payment is not made (100% coupon)...

I used 100% coupon and FS api threw an error saying it can't be empty...